src/monodroid/jni/config.h

```diff
@@ -4,8 +4,6 @@
 #include <monodroid.h>
 #include <stdint.h>
 
-MONO_API  int CreateNLSocket (void);
-MONO_API  int ReadEvents (void *sock, void *buffer, int32_t count, int32_t size);
-MONO_API  int CloseNLSocket (void *sock);
+// This file only exists for the include in zlib.c
```

Comment:
This kind of comment has a habit of becoming outdated with alarming frequency.

Reply:
Good point. I didn't think about it since I intend to get rid of the include mandating its presence, but it's probably worth making more generic for now.